src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSize.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Formats.Tar;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+[Trait("Category", "SdkContent")]
+public class ArtifactsSize : SmokeTests
+{
+    private readonly string baselineFilePath = BaselineHelper.GetBaselineFilePath($"ArtifactsSizes/{Config.TargetRid}.txt");
+    private readonly string[] baselineFileContent;
+    private readonly Regex buildVersionPattern = new(@"\b\d+\.\d+\.\d+[-@](alpha|preview|rc|rtm)\.\d(\.\d+\.\d+)?\b");
```

Comment:
nit: these can all be made static.

```suggestion
    private static readonly string baselineFilePath = BaselineHelper.GetBaselineFilePath($"ArtifactsSizes/{Config.TargetRid}.txt");
    private static readonly string[] baselineFileContent;
    private static readonly Regex buildVersionPattern = new(@"\b\d+\.\d+\.\d+[-@](alpha|preview|rc|rtm)\.\d(\.\d+\.\d+)?\b");
```

Reply:
For typical naming conventions in C# within .NET code, you would use a `_` prefix for fields (e.g. `_baselineFilePath`). For static fields it would have a `s_` prefix (`s_baselineFilePath`). And in cases like this where they're marked as `readonly`, they're effectively a constant, so I will often use the PascalCasing that would normally be used for constants (e.g. `BaselineFilePath`).

Reply:
Addressed [here](https://github.com/dotnet/installer/pull/17233/commits/4c601c036e6273a5b1d385811a7d0e16e99350df#diff-ce0a9d7a18f02288a0510a8ec86e3a5f387ad96613e0296f434de93f02c18da7L17).